### ISSUE TITLE
Support formatting of inline javascript in HTML files

### ISSUE THREAD

**Issue by M4urice** [jdc0589/JsFormat#156]
It would be nice to be able to format inline javascript in HTML files by selecting a code block and pressing a key combination or entering a command in the command palette.

**Comment by jdc0589** [comment]
i would be ok with this. you used to be able to do this,  but too many
people tried to format the whole file, which obviously screwed up html
content.

I *dont* want to start doing any html parsing right now though. So, we
would need to only allow formatting of highlighted code in html files.

**Comment by M4urice** [comment]
Well people should be able to decide themselves when to use the formatter, right (not talking about format-on-save)? If they do it wrong manually, undo is just a step away...

**Comment by jdc0589** [comment]
you are right. but I don't see a reason to ever allow formatting an entire
html file without a selection.
Ideally, we could parse out javascript tags from the current html file and
format all of those, but I'm not sure if I want to do that right now.

**Comment by M4urice** [comment]
>i would be ok with this. you used to be able to do this,  but too many
people tried to format the whole file, which obviously screwed up html
content.

How about just allowing it via the command palette then? 

[HTMLBeautify](https://github.com/rareyman/HTMLBeautify) allows formatting of select text like this:

<img width="1788" alt="screenshot 2016-05-24 15 32 04" src="https://cloud.githubusercontent.com/assets/1615089/15505673/b71d6246-21c4-11e6-93c3-0626aeff0c3a.png">

**Comment by jdc0589** [comment]
@M4urice I think I'm ok with that. It also solves the problem of quickly formatting some throw-away content being a 2-step process (set syntax, format).
